### PR TITLE
AdMob adapter 6.5.x implementation to support Banner Ad

### PR DESCRIPTION
**Pull request by YueVungle**
AdMob adapter 6.5.x version is based on Vungle iOS Monetization 6.5.x release and supports Banner Ads.
(Because I rename branch name yueting-sdk6.4.x  to 6.4.x, so the previous PR has been closed. And I reopen this PR, I'm so sorry for the inconvenience. ^_^ )

### CODE REVIEW

File: adapters/Vungle/VungleAdapter/VungleRouter.m [hunk 1]
```diff
@@ -101,22 +193,49 @@ - (NSError *)loadAd:(NSString *)placement withDelegate:(id<VungleDelegate>)deleg
...
+
 - (BOOL)playAd:(UIViewController *)viewController
       delegate:(id<VungleDelegate>)delegate
         extras:(VungleAdNetworkExtras *)extras {
   if (!delegate || !delegate.desiredPlacement ||
       ![[VungleSDK sharedSDK] isAdCachedForPlacementID:delegate.desiredPlacement]) {
     return false;
   }
-  NSMutableDictionary *options = [[NSMutableDictionary alloc] init];
+  NSMutableDictionary *options = [[NSMutableDictionary alloc] init];  //
```
Comment (crbinsf) [inline adapters/Vungle/VungleAdapter/VungleRouter.m]: remove `//`

File: adapters/Vungle/VungleAdapter/VungleRouter.m [hunk 2]
```diff
@@ -127,22 +246,82 @@ - (BOOL)playAd:(UIViewController *)viewController
...
+    return bannerView;
+  } else {
+    NSLog(@"Banner loading error: %@", bannerError.localizedDescription);
+    if (delegate.adapterAdType == MREC) {
+        self.isMrecPlaying = NO;
+    } else {
+        self.isBannerPlaying = NO;
+    }
+
+    return nil;
```
Comment (crbinsf) [inline adapters/Vungle/VungleAdapter/VungleRouter.m]: since the method will return `nil` is this needed?

**Comment by ashinagawa** [comment]
@YueVungle I noticed that some of `UnitIDs` in `ViewController` (which has not been changed in this PR tho), is set to "60d58da0b5324ae0b645670aa32104ab", "2b0d4aa5358c4190b9928601ea1f7a6a" and "de9b1104ee074542807e349f5a9ef2b5" are not belog to the same app as for other AdUnits. I think you need to prepare set of UnitIDs in the same app for all adtypes on MoPub dashboard, or this will confuse QA team.

**Comment by ashinagawa** [comment]
@YueVungle I also noticed that you use your banner UnitID is linked to Vungle adppid:5c003b9a3933314cf38ff7f3 which is used for Native iOS QA Test app.
I believe you need to prepare brand-new set of appID and placement ID just for MoPub QA Test app to avoid confusion. (since you only use 1 or 2 placements under "adppid:5c003b9a3933314cf38ff7f3" but, SDK tries to auto-cache placements which are not used in MoPub dashboard, so this is very confusing for QA team.)

File: adapters/Vungle/VungleAdapter/VungleRouter.m (hunk 1 above)
Comment (YueVungle) [reply adapters/Vungle/VungleAdapter/VungleRouter.m]: Got it.:)

File: adapters/Vungle/VungleAdapter/VungleRouter.m (hunk 2 above)
Comment (YueVungle) [reply adapters/Vungle/VungleAdapter/VungleRouter.m]: Got it, this “return nil” is useless and I will remove it.

**Comment by YueVungle** [comment]
> @YueVungle I also noticed that you use your banner UnitID is linked to Vungle adppid:5c003b9a3933314cf38ff7f3 which is used for Native iOS QA Test app.
> I believe you need to prepare brand-new set of appID and placement ID just for MoPub QA Test app to avoid confusion. (since you only use 1 or 2 placements under "adppid:5c003b9a3933314cf38ff7f3" but, SDK tries to auto-cache placements which are not used in MoPub dashboard, so this is very confusing for QA team.)
@ashinagawa Hi Aki, got it. Thanks a lot for the reminder. I will prepare brand-new set of appID and placement ID just for MoPub QA Test app.  😊

**Comment by ashinagawa** [comment]
@YueVungle Sorry I added my comments here which were about MoPub side. I moved them to MoPub side. Sorry for confusing.

**Pull request closed without merging**